### ISSUE TITLE
move error when no options 

### ISSUE THREAD

**Issue by benjaminroveri** [jprichardson/node-fs-extra#417]
```
Error:
TypeError: Cannot use 'in' operator to search for 'mkdirp' in undefined
at Object.move (/node_modules/fs-extra/lib/move/index.js:21:37)
```

Append when calling move without options: `fs.move( file_src, file_dst )`
It's OK this way: `fs.move( file_src, file_dst, {} )`

**Comment by manidlou** [comment]
@benjaminroveri what is your `fs-extra` version?

**Comment by manidlou** [comment]
Just a quick thing, all async functions get a callback function as their last parameter. I see that in your example, you are not using callback. With the latest version of `fs-extra`, you can use either promises or callbacks for async functions. It is also specified in [docs](https://github.com/jprichardson/node-fs-extra#sync-vs-async).

**Comment by RyanZim** [comment]
Closing for lack of action

**Comment by BiB-Apzx** [comment]
Hi,

This bug is still present:

```
TypeError: Cannot use 'in' operator to search for 'mkdirp' in undefined
    at Object.mv [as move] (C:\Users\BlocOdemo\git\api\node_modules\fs-extra\lib\move\index.js:19:32)
```

Adding an empty option object is solving this problem for now.

fs-extra version is `5.0.4`

**Comment by RyanZim** [comment]
@BiB-Apzx Your fs-extra version is out of date.

**Comment by BiB-Apzx** [comment]
@RyanZim Thanks, and now I feel stupid...

The bug is resolved in 7.0.1.
